platform/darwin/src/MGLStyle.mm

```diff
@@ -134,6 +132,35 @@ + (NSURL *)emeraldStyleURL {
     return MGLStyleURL_emerald;
 }
 
+// Traffic Day is no longer getting new versions as a default style, so the current version is hard-coded here.
+static NSURL *MGLStyleURL_trafficDay;
++ (NSURL *)trafficDayStyleURL {
+    static dispatch_once_t onceToken;
+    dispatch_once(&onceToken, ^{
+        MGLStyleURL_trafficDay = [NSURL URLWithString:@"mapbox://styles/mapbox/traffic-day-v2"];
+    });
+    return MGLStyleURL_trafficDay;
+}
+
++ (NSURL *)trafficDayStyleURLWithVersion:(NSInteger)version {
+    return [MGLStyle trafficDayStyleURL];
```

Comment:
`[MGLStyle trafficDayStyleURLWithVersion:1]` ends up returning Traffic Day v2. This could be a breaking change for any developers calling this method.

Reply:
@1ec5 I may misunderstood this but is it possible return a versioned style now that is deprecated? in the docs I wrote https://github.com/mapbox/mapbox-gl-native/pull/9918/files#diff-1d168de285fc4ba01ce48c4236f0b3ceR298

Reply:
`-trafficDayStyleURLWithVersion:` should return whatever version is specified, like it always has. `-trafficDayStyleURL` should return version 2. Both methods should be deprecated but should continue to work like they always have.